lib/containers/utils.pm

```diff
@@ -25,7 +25,7 @@ use version_utils;
 our @EXPORT = qw(test_seccomp basic_container_tests set_up get_vars build_img test_built_img);
 
 sub test_seccomp {
-    my $no_seccomp = script_run('docker info | tee /dev/$serialdev | grep seccomp');
+    my $no_seccomp = script_run('docker info | tee /dev/tty | grep seccomp');
```

Comment:
```suggestion
    my $no_seccomp = script_run('docker info | tee /tmp/docker_info.txt | grep seccomp');
    upload_logs('/tmp/docker_info.txt');
```
Wouldn't that be better than dealing with `$serialdev` on different platforms? 😉

Reply:
> Wouldn't that be better than dealing with `$serialdev` on different platforms?

Well, yes! We can go with this way :) I will commit and re-run the VRs